csrankings.csv

```diff
@@ -7205,9 +7205,9 @@ John Williamson,University of Glasgow,http://www.dcs.gla.ac.uk/~jhw,fcvb8f8AAAAJ
 John Yiannis Cotronis,University of Athens,http://www.di.uoa.gr/eng/staff/1022,9IbOFmMAAAAJ
 John Zahorjan,University of Washington,https://homes.cs.washington.edu/~zahorjan,NOSCHOLARPAGE
 John Zimmerman,Carnegie Mellon University,https://www.cs.cmu.edu/~johnz,A-4JUL4AAAAJ
-Johnny S. Wong,Iowa State University,http://www.cs.iastate.edu/people/johnny-wong,w3UeCAoAAAAJ
-Johnny Wong,Iowa State University,http://www.cs.iastate.edu/people/johnny-wong,w3UeCAoAAAAJ
-Johnson Thomas,Oklahoma State University,http://www.cs.okstate.edu/~jpt,pyIZFggAAAAJ
+Johnny S. Wong,Iowa State University,http://www.cs.iastate.edu/people/johnny-wong,57pKMKwAAAAJ
+Johnny Wong,Iowa State University,http://www.cs.iastate.edu/people/johnny-wong,57pKMKwAAAAJ

```

Comment:
Another randomly selected case where both are wrong: neither the original (w3UeCAoAAAAJ) nor the replacement (57pKMKwAAAAJ) are correct.